src/Nelmio/Alice/Fixtures/Parser/Methods/Base.php

```diff
@@ -39,7 +39,7 @@ public function canParse($file)
     {
         // we add (\.php)? to the regex to allow extensions of this parser to first
         // be compiled by php
-        return preg_match("/\.{$this->extension}(\.php)?$/", $file) == 1;
+        return 1 === preg_match("/\\.{$this->extension}(\\.php)?$/", $file);
```

Comment:
Backslash is used to escape special characters, thou there is no \. sequence, it's safier to use doubled backslash
> To specify a literal backslash, double it (\\)

See double quotes string type docs - http://php.net/manual/en/language.types.string.php#language.types.string.syntax.double